modules/bridge/src/main/ts/ephox/bridge/components/menu/CardMenuItem.ts

```diff
@@ -0,0 +1,45 @@
+import { FieldSchema, ValueSchema } from '@ephox/boulder';
+import { Fun, Result } from '@ephox/katamari';
+import { Autocomplete, AutocompleteSpec } from './Autocomplete';
+import { CommonMenuItem, commonMenuItemFields, CommonMenuItemInstanceApi, CommonMenuItemSpec } from './CommonMenuItem';
+import { Container, ContainerSpec, itemSchema } from './Container';
+import { Description, DescriptionSpec } from './Description';
+import { Image, ImageSpec } from './Image';
+
+export type CardItemsSpec =
```

Comment:
This is missing `Menu` in the name, so it's inconsistent. It should probably be `CardMenuItemsSpec`.

Reply:
Update: Ignore this, as we said we should rename this to be more generic (see the comment on `Container.ts`).